server/meca/routes.js

```diff
@@ -37,23 +37,37 @@ module.exports = app => {
 
     const updateStatus = () => Manuscript.updateStatus(manuscriptId, status)
 
-    const sendEmail = () => mailer.send({
-      to: config.get('meca.email.recipient'),
-      from: config.get('meca.email.sender'),
-      subject: `${config.get('meca.email.subjectPrefix')}MECA import failed`,
-      text: `
+    const auditlog = () =>
+      new AuditLog({
+        action: 'UPDATED',
+        objectId: manuscriptId,
+        objectType: 'ejp.id',
+        value: body.id,
+      }).save()
+
+    const sendEmail = () =>
+      mailer
+        .send({
+          to: config.get('meca.email.recipient'),
+          from: config.get('meca.email.sender'),
+          subject: `${config.get(
+            'meca.email.subjectPrefix',
+          )}MECA import failed`,
+          text: `
 EJP failed to import MECA package.
 Manuscript ID: ${manuscriptId}
       `,
-    }).catch((err) => {
-      throw new Error(`MECA import failure email failed to send. ${err}`)
-    })
+        })
+        .catch(err => {
+          throw new Error(`MECA import failure email failed to send. ${err}`)
+        })
 
     updateStatus()
       .then(() => {
         if (status === Manuscript.statuses.MECA_IMPORT_FAILED) {
           return sendEmail()
         }
+        auditlog()
```

Comment:
I don't think the audit will happen if the mailer above throws an error? Should this be done before in its own `.then`?

Reply:
Didn't realise this was a draft so you may have picked up on this already. Can you put `WIP:` in the title still so it's a bit clearer on the PR page?

Reply:
It's ok good feedback, anyway!